core/src/Streamly/Internal/Data/Unfold.hs

```diff
@@ -237,7 +237,7 @@ module Streamly.Internal.Data.Unfold
     , crossWithM
     , crossWith
     , cross
-    , apply
+    , crossApply
```

Comment:
`cross` vs `nest`

Reply:
nest is a very general name, cross implies cross product which is exactly what it is.